ndcube/ndcube.py

```diff
@@ -107,6 +108,29 @@ def crop_by_coords(self, lower_left_corner, dimension_widths):
 
         """
 
+    @abc.abstractmethod
+    def crop_by_extra_coord(self, min_coord_value, interval_width, extra_coord):
+        """
+        Crops an NDCube given a minimum value and interval width along an extra coord.
+
+        Parameters
+        ----------
+        min_coord_value: Single value of type consistent with the extra coord.
```

Comment:
Should the *type* here be `Quantity`? and the unit be consistent with the extra_coord?

Reply:
Yes!  Done.